test/WorkerExtensionTests/WorkerExtensionTests.csproj

```diff
@@ -24,7 +24,23 @@
     <ItemGroup>
         <ProjectReference Include="..\..\extensions\Worker.Extensions.Storage\src\Worker.Extensions.Storage.csproj" />
         <ProjectReference Include="..\..\extensions\Worker.Extensions.CosmosDB\src\Worker.Extensions.CosmosDB.csproj" />
+        <ProjectReference Include="..\..\extensions\Worker.Extensions.Tables\src\Worker.Extensions.Tables.csproj" />
         <ProjectReference Include="..\DotNetWorkerTests\DotNetWorkerTests.csproj" />
     </ItemGroup>
 
+    <ItemGroup>
+
+            <ProjectReference Include="..\..\extensions\Worker.Extensions.Abstractions\src\Worker.Extensions.Abstractions.csproj" />
+
+            <ProjectReference Include="..\..\extensions\Worker.Extensions.Http\src\Worker.Extensions.Http.csproj" />
+
+            <ProjectReference Include="..\..\extensions\Worker.Extensions.Storage\src\Worker.Extensions.Storage.csproj" />
+
+            <ProjectReference Include="..\..\src\DotNetWorker.ApplicationInsights\DotNetWorker.ApplicationInsights.csproj" />
+
+            <ProjectReference Include="..\..\src\DotNetWorker\DotNetWorker.csproj" />
+
+         
```

Comment:
This looks like a duplicate item group for project references, we already have a lot of these imported on lines 25-28. Remove this section and update the one above it to add what you're missing if anything